src/DynamoCore/Search/SearchElements/NodeSearchElement.cs

```diff
@@ -283,6 +283,14 @@ protected virtual IEnumerable<string> GenerateOutputParameters()
             inputParameters.Add(Tuple.Create(String.Empty, Properties.Resources.NoneString));
             return inputParameters;
         }
+
+        /// <summary>
+        ///    Clone the NodeSearchElement.
+        /// </summary>
+        public object Clone()
```

Comment:
internal?

Reply:
Has to be public as NodeSearchElement is extending ICloneable.